backend/src/index.js

```diff
@@ -71,8 +71,9 @@ app.use(APIErrorHandler)
 /*
  * Connect to SQL server
  */
-db.init()
-    .then(() => console.log('App ready!'))
-    .then(() => app.emit('ready'))
+db.on('ready', () => {
```

Comment:
where does this ready event come from?

Reply:
the database emits ready when the DB class's constructor successfully connects to the database